src/mlpack/methods/ann/loss_functions/margin_ranking_loss.hpp

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file margin_ranking_loss.hpp
+ * @author Andrei Mihalea
+ *
+ * Definition of the Margin Ranking Loss function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_ANN_LOSS_FUNCTION_MARGIN_RANKING_LOSS_HPP
+#define MLPACK_ANN_LOSS_FUNCTION_MARGIN_RANKING_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+template <
```

Comment:
Hi @AndreiMihalea, Could you also please add template parameters description. They go along the line of : 
```
@tparam InputDataType Type of the input data (arma::colvec, arma::mat, arma::sp_mat or arma::cube).
```
You might need to wrap this in two lines as mlpack limits each line to 80 characters.